# This message specifies a bounding box for CheckKinematics service.

# Only SolidPrimitive.type = BOX is supported.
shape_msgs/SolidPrimitive box
# The pose of the bounding box in the robot base frame.
geometry_msgs/Pose pose
